### PR TITLE
Add Airlock

### PR DESCRIPTION
**Pull request by emm1R**
User can now export files from SD Desktop to SD Connect using the Data Gateway UI

### CODE REVIEW

**Review by blankdots — COMMENTED** [review]
might want to update changelog as well.

File: README.md [hunk 1]
```diff
@@ -25,7 +25,8 @@ Set these environment variables before running the application:
 - `FS_SD_CONNECT_API` - API for SD-Connect
 - `FS_SD_SUBMIT_API` – a comma-separated list of APIs for SD Apply/SD Submit
 - `SDS_ACCESS_TOKEN` - a JWT for authenticating to the SD APIs
-- `FS_CERTS` is the path to a file that contains certificates required by SD Connect and SD Apply/SD Submit
+- `FS_CERTS` - path to a file that contains certificates required by SD Connect, SD Apply/SD Submit, and SDS AAI 
+- `USERINFO_ENDPOINT` - endpoint in SDS AAI that reveals which projects the user is a manager of
```
Comment (blankdots) [inline README.md:29]: i think we can get this endpoint from this file: `/etc/pam_userinfo/config.json'` it contains: `userinfo_endpoint` we add it by default.

Comment (emm1R) [reply README.md:29]: Ok. The USERINFO_ENDPOINT was in the vm already though.

Comment (blankdots) [reply README.md:29]: yes, not sure if anything else uses that, we can leave it.
or if we want to be volatile we first try to read it from env var first and if that does not exist we fall back to reading from file.

**Pull request merged**